lib/internal/process/next_tick.js

```diff
@@ -137,20 +137,13 @@ function setupNextTick() {
     } while (tickInfo[kLength] !== 0);
   }
 
-  function nextTick(callback) {
+  function nextTick(callback, ...args) {
     if (typeof callback !== 'function')
       throw new TypeError('callback is not a function');
     // on the way out, don't bother. it won't get fired anyway.
     if (process._exiting)
       return;
 
-    var args;
```

Comment:
There is a minor difference now when `arguments.length === 1` as `args` is an empty array, but I don't know if it's relevant. If not LGTM.